src/main/java/com/arpnetworking/tsdcore/model/PeriodicDataToProtoConverter.java

```diff
@@ -134,4 +140,8 @@ private static ByteString serializeSupportingData(final AggregatedData datum) {
         }
         return byteString;
     }
+
+    private PeriodicDataToProtoConverter() {
+        throw new AssertionError("can't construct utility class");
```

Comment:
You don't need to throw in here. Just marking it private is enough to prevent someone from instantiating it.